exe/Main.hs

```diff
@@ -48,3 +107,9 @@ main = do
         ( fullDesc
             <> progDesc "Run Tailwind"
         )
+    readPlugins txt =
+      let ePlugins = for (Text.splitOn "," txt) \strPlugin ->
+            case readMaybe $ Text.unpack $ Text.toLower $ Text.strip strPlugin of
+              Just plugin -> pure plugin
+              Nothing     -> Left $ "Could not parse " <> strPlugin
+      in  either error id ePlugins
```

Comment:
You can actually do the parsing (and error handling) as part of CLI argument parsing. Example: https://github.com/srid/rib/blob/db26bcde09364849683736bc1dc30acdff5d6cc0/rib-core/src/Rib/Cli.hs#L108